packages/playground/_stories/main/ComboBox/ComboBox.stories.ts

```diff
@@ -74,30 +68,32 @@ Filters.args = {
 		<ui5-cb-item text="Austria"></ui5-cb-item>
 		<ui5-cb-item text="Bulgaria"></ui5-cb-item>
 		<ui5-cb-item text="Germany"></ui5-cb-item>
-		<ui5-cb-item text="United Kingdom"></ui5-cb-item>
 		<ui5-cb-item text="Kazakhstan"></ui5-cb-item>
+		<ui5-cb-item text="The United Kingdom of Great Britain and Northern Ireland"></ui5-cb-item>
+		<ui5-cb-item text="United Kingdom"></ui5-cb-item>
 	`,
 };
 
-export const TwoColumns = Template.bind({});
-TwoColumns.args = {
+export const TwoColumnsLayout = Template.bind({});
+TwoColumnsLayout.args = {
 	placeholder: "Two-column layout",
 	default: `
 		<ui5-cb-item text="Austria" additional-text="AT"></ui5-cb-item>
 		<ui5-cb-item text="Belgium" additional-text="BE"></ui5-cb-item>
 		<ui5-cb-item text="Brazil" additional-text="BR"></ui5-cb-item>
 		<ui5-cb-item text="Bulgaria" additional-text="BG"></ui5-cb-item>
 		<ui5-cb-item text="Canada" additional-text="CA"></ui5-cb-item>
+		<ui5-cb-item text="The United Kingdom of Great Britain and Northern Ireland" additional-text="UK"></ui5-cb-item>
 	`,
 };
 
-export const GroupingItems = Template.bind({});
-GroupingItems.args = {
-	placeholder: "ComboBox with grouping of suggestions",
+export const Grouping = Template.bind({});
+Grouping.args = {
+	placeholder: "Grouping of suggestions",
```

Comment:
I am not sure if suggestions is the right word here. ComboBox provides list of Options (Items technically).